Makefile

```diff
@@ -30,6 +30,20 @@ lint: check_pylint_installed check_pytest_installed
 .PHONY: test
 test: check_pytest_installed
 	@pytest -x -v --ignore=third_party/ --ignore=tests/e2e_test --disable-warnings
+	@python examlpes/offline_inference.py
+	@pytest -v tests/e2e_test/test_e2e.py
+
+.PHONY: unit_test
+unit_test: check_pytest_installed
+	@pytest -x -v --ignore=third_party/ --ignore=tests/e2e_test --disable-warnings
+	
+.PHONY: offline_test
+offline_test:
+	@python examlpes/offline_inference.py
+
+.PHONY: e2e_test
+e2e_test:
+	@pytest -v tests/e2e_test/test_e2e.py
```

Comment:
I suggest to delete e2e_test and offline_test in Makefile and revert the change in .github/workflows/offline_inference.yml.

Generally, we can just run the e2e_test and offline_test  through python command.
Later, I will add a make command to run all the tests

Reply:
Get